Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM pcic/geospatial-python
+
+ADD . /app
+WORKDIR /app
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && \
+    apt-get -yq install postgresql postgresql-contrib postgis postgresql-9.3-postgis-2.1
```

Comment:
Also, should we update `pcic/geospatial-python` to use Postgres 9.5, as our dev environments do? Then we'd have to install `postgresql-9.5-postgis-2.2`